### PR TITLE
MSPSDS-760: Service tidy up

### PR DESCRIPTION
**Pull request by aidayounesi**
- Update header terminology
- Delete product/business merge
- Check for usage of term 'investigation' and replace by 'case'
- Change ‘enquiry’ to ‘question': terminology consistent throughout

### CODE REVIEW

File: web/app/views/businesses/index.html.slim [hunk 1]
```diff
@@ -17,7 +17,6 @@ main#main-content.govuk-main-wrapper role="main"
             = form.submit "Search", name: nil, class: "mspsds-search-box--submit"
     .govuk-grid-column-three-quarters
       = form_with(url: confirm_merge_businesses_path, method: :get) do |form|
```
Comment (tom-bell-softwire) [inline web/app/views/businesses/index.html.slim]: I think we probably want to remove this form as well, and the `form` elements in the "table" partial below.
(These added the checkboxes next to each table entry that were used as part of the merge process.)

File: web/app/views/products/index.html.slim [hunk 2]
```diff
@@ -17,7 +17,6 @@ main#main-content.govuk-main-wrapper role="main"
             = form.submit "Search", name: nil, class: "mspsds-search-box--submit"
     .govuk-grid-column-three-quarters
       = form_with(url: confirm_merge_products_path, method: :get) do |form|
```
Comment (tom-bell-softwire) [inline web/app/views/products/index.html.slim]: Same here – we should remove the form and its use in the "table" partial below.

File: web/app/views/layouts/application.html.slim [hunk 3]
```diff
@@ -57,26 +57,26 @@ html.govuk-template.app-html-class[lang="en"]
...
             ul#navigation.govuk-header__navigation[aria-label="Top Level Navigation"]
               - if user_signed_in?
                 li.govuk-header__navigation-item
-                  = link_to "Manage your account", user_account_url, class: "govuk-header__link", target: "_blank"
+                  = link_to "Your account", user_account_url, class: "govuk-header__link", target: "_blank"
               - if !login_page?
                 li.govuk-header__navigation-item
                   - if user_signed_in?
-                    = link_to "Logout", logout_session_path, method: :delete, class: "govuk-header__link"
+                    = link_to "Sign out", logout_session_path, method: :delete, class: "govuk-header__link"
```
Comment (tom-bell-softwire) [inline web/app/views/layouts/application.html.slim]: I think we have a couple of notification banners that say "Logged in successfully" / "Logged out successfully" that we should probably also update.

File: web/app/controllers/businesses_controller.rb [hunk 4]
```diff
@@ -22,31 +22,6 @@ def show
     CompaniesHouseClient.instance.update_business_from_companies_house(@business)
   end
 
-  # GET /businesses/confirm_merge
-  def confirm_merge
```
Comment (tom-bell-softwire) [inline web/app/controllers/businesses_controller.rb]: Can you also remove the associated routes for these endpoints?

Comment (aidayounesi) [reply web/app/controllers/businesses_controller.rb]: Sure, done

File: web/app/views/layouts/application.html.slim (hunk 3 above)
Comment (aidayounesi) [reply web/app/views/layouts/application.html.slim]: Thanks, Done

File: web/app/views/businesses/index.html.slim (hunk 1 above)
Comment (aidayounesi) [reply web/app/views/businesses/index.html.slim]: Sure, done, thanks

File: web/app/views/products/index.html.slim (hunk 2 above)
Comment (aidayounesi) [reply web/app/views/products/index.html.slim]: Done

File: web/config/routes.rb [hunk 5]
```diff
@@ -24,7 +24,8 @@
     end
   end
 
-  resources :investigations, only: %i[index show new create update], concerns: %i[document_attachable image_attachable] do
+  resources :cases, controller: "investigations", as: "investigations",
```
Comment (broder) [inline web/config/routes.rb]: Do you need both `controller: "investigations"` and `as: "investigations"`?

**Comment by aidayounesi** [comment]
Thanks

**Pull request merged**